### DISCUSSION TITLE
Difference between this and other free open source upscale tools?

### DISCUSSION THREAD

**Discussion by deroverda** [upscayl/upscayl#764] [Q&A]
Hi guys, 

I just downloaded the portable win version and this looks pretty damn good. I've been running Waifu2x-Extension-GUI (by AaronFeng753) since 1 year back and it has worked good. Just simply wondering what are the pros cons using this compared waifu2x? 

Thanks in advance.

**Comment by aaronliu0130** [comment]
Differences:

* WEG has a lot more features and algorithms
* WEG uses Qt, which uses native widgets, instead of Electron, which uses HTML widgets
* WEG looks uglier
* We have a free license while Waifu2x doesn't

Similarities:

* We both have a Chinese member named Aaron!
* We are both GUIs for various CLIs
* We both support the REAL-ESRGAN algorithm and basic image processing

**Reply by NayamAmarshe** [reply]
Some more:
- WEG is **Windows ONLY**. Upscayl supports Mac, Linux and Windows and soon every browser out there.
- WEG is for technically inclined people, Upscayl focuses on simplicity and a better user experience.
- WEG is paid for commercial use, we're not.

You can try both and then choose, of course 😄
